operator/v1/types_klusterlet.go

```diff
@@ -86,6 +86,24 @@ type KlusterletSpec struct {
 	// registration-agent and work-agent will use it to communicate with hub api server.
 	// +optional
 	HubApiServerHostAlias *HubApiServerHostAlias `json:"hubApiServerHostAlias,omitempty"`
+
+	// ResourceRequirement specify QoS classes of klusterlet deployment
+	// +optional
+	ResourceRequirement *ResourceRequirement `json:"resourceRequirement,omitempty"`
+}
+
+type ResourceQosClass string
+
+const (
+	ResourceQosClassDefault    ResourceQosClass = "Default"
+	ResourceQosClassBestEffort ResourceQosClass = "BestEffort"
```

Comment:
some comments maybe to indicate what besteffort means

Reply:
comments added